modules/multi-architecture-verifying-cluster-compatibility.adoc

```diff
@@ -18,6 +23,18 @@ Before you can start adding compute nodes of different architectures to your clu
 
 * You installed the OpenShift CLI (`oc`)
 
+ifdef::ibm-power[]
+[NOTE]
+====
+When using multiple architectures, hosts for the {product-title} nodes must share the same storage layer. If they do not have the same storage layer, please use a storage provider such as `nfs-provisioner`.
```

Comment:
I'm going to suggest something that the team has talked about, but that there is no written rule for (other than saying not to "overuse admonitions." https://redhat-documentation.github.io/supplementary-style-guide/#admonitions. So consider this change, but I won't hold up another merge-review because of it. 

In the past, we've tried to stay away from stacking admonitions in favor of having 1 admonition with 2 bullet points. Because there's no rule on this I usually wouldn't hold it up for merge review, but "please" needs to be removed in this note as well, as per IBMSG: I'm going to suggest something that the team has talked about, but that there is no written rule for (other than saying not to "overuse admonitions." https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

In the past, we've tried to stay away from stacking admonitions in favor of having 1 admonition with 2 bullet points. Because there's no rule on this I usually wouldn't hold it up for merge review, but "please" needs to be removed in this note as well: 

```suggestion
When using multiple architectures, hosts for the {product-title} nodes must share the same storage layer. If they do not have the same storage layer, use a storage provider such as `nfs-provisioner`.
```

So, I'm requesting the removal of please, and the consideration of using bullet points in one NOTE instead of stacking 2 notes.